### ISSUE TITLE
String.format causes crash during initialization of case object

### ISSUE THREAD

**Issue by frgomes** [scala-native/scala-native#904]
```
scala.NotImplementedError: an implementation is missing
        at java.lang.Throwable.init(Unknown Source)
        at java.lang.Error.init(Unknown Source)
        at java.lang.Error.init(Unknown Source)
        at scala.NotImplementedError.init(Unknown Source)
        at scala.NotImplementedError.init(Unknown Source)
        at scala.Predef$.???(Unknown Source)
        at java.util.Formatter.format(Unknown Source)
        at java.lang.String$.format(Unknown Source)
        at java.lang.String$.format(Unknown Source)
```
More details at https://github.com/frgomes/scala-bindgen/issues/32

**Comment by matil019** [comment]
`java.util.Formatter`, which `String.format` depends on, is not implemented yet. See #804 for details.

**Comment by densh** [comment]
Fixed in #804
